packages/compass-crud/src/stores/crud-store.js

```diff
@@ -717,16 +717,22 @@ const configureStore = (options = {}) => {
      * Emits a global app registry event the plugin listens to.
      */
     openImportFileDialog() {
-      this.localAppRegistry.emit('open-import');
+      this.globalAppRegistry.emit('open-import', { namespace: this.state.ns });
     },
 
     /**
      * Open an export file dialog from compass-import-export-plugin.
      * Emits a global app registry event the plugin listens to.
      */
     openExportFileDialog() {
-      // Pass the doc count to the export modal so we can avoid re-counting.
-      this.localAppRegistry.emit('open-export', this.state.count);
+      // Only three query fields that export modal will handle
+      const { filter, limit, skip } = this.state.query;
+      this.globalAppRegistry.emit('open-export', {
```

Comment:
Nicee looks like this fixes https://jira.mongodb.org/browse/COMPASS-5143 too? I pulled it into sprint if that's cool.

Reply:
Haha, I didn't even know this was a bug before, yeah, seems to be fixed, thanks for spotting